addons/weather/functions/fnc_displayWindInfo.sqf

```diff
@@ -85,23 +90,26 @@ TRACE_1("Starting Wind Info PFEH", GVAR(WindInfo));
     };
     __ctrl ctrlCommit 0;
 
-    //Update the beaufort balls:
-    (ctrlPosition __ctrl) params ["_ctrlX", "_ctrlY", "_ctrlWidth", "_ctrlHeight"];
-    private _centerX = _ctrlX + _ctrlWidth / 2;
-    private _centerY = _ctrlY + _ctrlHeight / 2;
-    private _ballHeight = _ctrlHeight / 17;
-    private _ballWidth = _ballHeight * 3/4;
-
-    for "_index" from 0 to (_beaufortNumber - 1) do {
-        private _ball = __dsp ctrlCreate ["RscPicture", _index];
-        _ball ctrlSetText QPATHTOF(UI\wind_dot_ca.paa);
-        _ball ctrlSetTextColor [1,1,1,1];
-        private _ballCenterX = _centerX - (_ballWidth / 2) + ((sin _windDir) * 0.013333) * (_index - 4.9) + ((cos _windDir) * 0.0125);
-        private _ballCenterY = _centerY - (_ballHeight / 2) - ((1 * cos _windDir) * 4/3*0.013333) * (_index - 4.9) + ((sin _windDir) * 0.0125);
-        _ball ctrlSetPosition [_ballCenterX, _ballCenterY, _ballWidth, _ballHeight];
-        _ball ctrlCommit 0;
+    if (!_isUsingChute) then {
```

Comment:
looks like `exitWith` could be used here for early exit instead of big `then` block